tests/src/Fixtures/TripalImporter/TestTripalImporter.php

```diff
@@ -0,0 +1,606 @@
+      $container->get('tripal_chado.database'),
+      $container->get('plugin.manager.trpcultivate_validator'),
+      $container->get('entity_type.manager'),
+      $container->get('messenger'),
+    );
+  }
+
+  /**
+   * {@inheritDoc}
+   *
+   * THIS METHOD IS REQUIRED!
+   */
+  public function configureValidators(array $form_values, string $file_mime_type) {
+
+    $validators = [];
+
+    // CONFIGURE VALIDATOR HERE.
+    // ..................................................
+    // SEE src/Plugins/Validators for VALIDATOR IDs.
+    // SEE TripalCultivateValidator/ValidatorTraits for setters and getters.
+    //
+    // CONFIGURATION TEMPLATE:
+    // $id = 'REPLACE WITH VALIDATOR ID ANNOTATION - @id';
+    $id = 'valid_delimited_file';
+
+    // $input_type = 'REPLACE WITH VALIDATOR INPUT TYPE - @input_types';
+    $input_type = 'raw-row';
+
+    $instance = $this->service_validatorPluginManager->createInstance($id);
+    //
+    // Use relevant setters to set some values.
+    // For example:
+    // $instance->setExpectedColumns(count($this->headers), TRUE);
+    // $instance->setFileMimeType($file_mime_type);
+    // $instance->setHeaders($this->headers);
+    // $instance->setIndices(1);
+    //
+    $instance->setExpectedColumns(count($this->headers), TRUE);
+    $instance->setFileMimeType($file_mime_type);
+
+    // Register the validator instance.
+    $validators[$input_type][$id] = $instance;
+
+    return $validators;
+  }
+
+  /**
+   * {@inheritDoc}
+   *
+   * THIS METHOD IS REQUIRED!
+   */
+  public function processValidationMessages($failures) {
+
+    $messages = [];
+    // CALL PROCESS MESSAGE HERE.
+    // ..................................................
+    // SEE src/Plugins/Validators for VALIDATOR IDs.
+    //
+    // PROCESS MESSAGE TEMPLATE:
+    // 1: Create a $messages entry.
+    // $id = 'REPLACE WITH VALIDATOR ID ANNOTATION - @id';
+    $id = 'valid_delimited_file';
+
+    // Use title key to set a validator case message in $messages array.
+    // $messages[$id] = ['title' => 'REPLACE WITH A MESSAGE TITLE', ...];.
+    $messages[$id] = [
+      'title' => 'File is delimited',
+      'status' => 'todo',
+      'details' => '',
+    ];
+
+    // 2: Inspect validator-specific $failures and switch status accordingly.
+    if (array_key_exists($id, $failures)) {
+      if (!empty($failures[$id])) {
+        $messages[$id]['status'] = 'fail';
+        // Use the message processor static method defined by the validator.
+        // $messages[$id]['details']='ValidatorClassName::processMessageName()';
+        //
+        // or if the method is defined in this class, call the method as shown
+        // in the following line.
+        $messages[$id]['details'] = $this->processValidDelimitedFileFailures($failures[$id]);
+      }
+      else {
+        $messages[$id]['status'] = 'pass';
+      }
+    }
+
+    return $messages;
+  }
+
+  /**
+   * {@inheritdoc}
+   *
+   * THIS METHOD IS REQUIRED!
```

Comment:
With the new updates, there should be no need to make changes here. As such I suggest we update it to match the other methods where no changes are needed.

```suggestion
   * DO NOT MODIFY.
```